src/command_modules/azure-cli-role/azure/cli/command_modules/role/generated.py

```diff
@@ -34,15 +36,21 @@
 cli_command('role assignment create', create_role_assignment, simple_output_query=simple_output_query)
 
 factory = lambda _: _graph_client_factory().applications
-cli_command('ad app delete', ApplicationsOperations.delete, factory)
-cli_command('ad app show', ApplicationsOperations.get, factory)
+cli_command('ad app create', create_application, factory)
+cli_command('ad app delete', delete_application, factory)
 cli_command('ad app list', list_apps, factory)
+cli_command('ad app show', show_application, factory)
```

Comment:
I am working to provide a query for the tab output, and will push out the change soon